step.yml

```diff
@@ -0,0 +1,123 @@
+title: AppCenter Android Deploy
+summary: Deploy your awesome Android app to AppCenter.
+description: |-
+  Deploy an Android application to [AppCenter](https://appcenter.ms).
+
+  If an `app_id` is provided, the APK file will be uploaded to the specified app on HockeyApp. 
+  Otherwise HockeyApp will decide whether it's a new app or an update to an existing app based on the package ID.
+website: https://github.com/bitrise-steplib/steps-appcenter-android
+source_code_url: https://github.com/bitrise-steplib/steps-appcenter-android
+support_url: https://github.com/bitrise-steplib/steps-appcenter-android/issues
+
+project_type_tags:
+  - android
+  - xamarin
+  - cordova
+  - ionic
+  - react-native
+type_tags:
+  - deploy
+
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-hockeyapp-android-deploy
+
+inputs:
+  - apk_path: "$BITRISE_APK_PATH"
+    opts:
+      title: APK path
+      summary: Path to binary file
+      description: Path to binary file
+      is_required: true
+  - mapping_path:
+    opts:
+      title: mapping.txt file path
+      summary: Path to an Android mapping.txt file.
+      description: Path to an Android mapping.txt file.
+  - api_token:
+    opts:
+      title: API Token
+      summary: App Center API token
+      description: App Center API token
+      is_required: true
+      is_sensitive: true
+  - owner_name:
+    opts:
+      title: Owner name
+      summary: Owner of the App Center app
+      description: Owner of the App Center app
+      is_required: true
+  - app_name:
+    opts:
+      title: App name
+      summary: The name of the App Center app
+      description: The name of the App Center app
+      is_required: true
+  - distribution_group:
+    opts:
+      title: Distribution groups
+      summary: User groups you wish to distribute the app. One name per line.
```

Comment:
User groups you wish to distribute the app. One name per line. this two sentence makes a strange feeling in me. please somehow make it more understandable